### DISCUSSION TITLE
Private Fabric Port

### DISCUSSION THREAD

**Discussion by S-moreo** [P3pp3rF1y/SophisticatedBackpacks#398] [Q&A]
Hello! I was looking for an entry level project for getting into the modding community. I'm currently hosting a private forge server of less than 10 people, and we're currently in the process of moving to fabric. We've all fell in love with your mod, and I was wondering if I could create a fabric port. The mod would remain private, and would never be released, posted, or shared beyond our small community w/o permission. 

TL;DR : I would like your permission to use your visual assets for a private fabric port.

**Comment by P3pp3rF1y** [comment]
I can double check on the visual assets as they are not really mine technically, but to be honest this is not really a good modding entry level project. I would say if you're really good with Java it may be ok, but still there are way easier projects to get into. There's just a lot going on in the mod itself and it's pretty easy to break it and get weird issues if you don't understand how the logic specifically the upgrade one works.

**Comment by g-lewis** [comment]
> Hello! I was looking for an entry level project for getting into the modding community. I'm currently hosting a private forge server of less than 10 people, and we're currently in the process of moving to fabric. We've all fell in love with your mod, and I was wondering if I could create a fabric port. The mod would remain private, and would never be released, posted, or shared beyond our small community w/o permission.
> 
> TL;DR : I would like your permission to use your visual assets for a private fabric port.

If you did ever make one, I'd love to know, I'm set on using Fabric and would love to use this mod.
